_1327/minutes/models.py

```diff
@@ -12,6 +13,31 @@
 MINUTES_VIEW_PERMISSION_NAME = 'view_minutesdocument'
 
 
+class HexColorModelField(models.CharField):
+
+	def __init__(self, *args, **kwargs):
+		kwargs['max_length'] = 7
+		super(HexColorModelField, self).__init__(*args, **kwargs)
+
+	default_validators = [RegexValidator("^#(?:[0-9a-fA-F]{3}){1,2}$")]
+
+
+class MinutesLabel(models.Model):
+	title = models.CharField(max_length=255)
+	color = HexColorModelField()
+
+	def __str__(self):
+		return self.title
+
+	@property
+	def text_color(self):
+		r = int(self.color[1:3], 16)
+		g = int(self.color[3:5], 16)
+		b = int(self.color[5:7], 16)
+		yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000
+		return '#171717' if (yiq >= 128) else '#ebeced'
```

Comment:
Wouldn't it make more sense to use bootstrap classes instead of hardcoded colors?